recipes/pyrser/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pyrser" %}
+{% set version = "2.5.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/LionelAuroux/{{ name }}/archive/master.tar.gz
```

Comment:
This needs to not just be a snapshot off master, it really needs to be some kind of tagged version.

Reply:
Switched it to use the wheel file found in pypi https://pypi.org/project/pyrser/#files